aspnetcore/blazor/components.md

```diff
@@ -142,7 +142,7 @@ When the component is rendered, the `value` of the input element comes from the
 In addition to `onchange`, the property can be bound using other events like `oninput` by adding a `@bind` attribute with an `event` parameter:
 
 ```cshtml
-<input type="text" @bind-value="@CurrentValue" @bind-value:event="oninput" />
+<input @bind-value="@CurrentValue" @bind-value:event="oninput" />
```

Comment:
Why was `type="text"` removed?

Reply:
I thought everyone was allowing the default `type` to take care of it (e.g., Dan did it that way in the current ToDo List tutorial; Steve in the Blazing Pizza workshop). We're leaning that way, too, with the upcoming changes to the *Build your first* experience. I was just trying to make everything consistent. Is there a reason to keep it?

Reply:
I see. Okay